src/status_im/contexts/onboarding/intro/view.cljs

```diff
@@ -19,9 +20,32 @@
   []
   (rf/dispatch [:show-bottom-sheet {:content privacy/privacy-statement :shell? true}]))
```

Comment:
Leaving the comment here, but unrelated to this line of code. If privacy mode is enabled, we shouldn't ask the user for usage data collection in the onboarding because that would violate the toggle's behavior.